src/databased/runtime.rs

```diff
@@ -504,37 +504,31 @@ impl Database {
 
     fn get_all_bitcoin_addresses(
         &mut self,
-    ) -> Result<Vec<(bitcoin::Address, Option<SwapId>)>, lmdb::Error> {
+    ) -> Result<Vec<(bitcoin::Address, Option<SwapId>)>, Error> {
         let db = self.0.open_db(Some(LMDB_BITCOIN_ADDRESSES))?;
         let tx = self.0.begin_ro_txn()?;
         let mut cursor = tx.open_ro_cursor(db)?;
         let res = cursor
             .iter()
-            .filter_map(|(key, val)| {
-                bitcoin::Address::strict_decode(std::io::Cursor::new(key.to_vec()))
-                    .ok()
-                    .map(|a| {
-                        (
-                            a,
-                            BitcoinSecretKeyInfo::strict_decode(std::io::Cursor::new(val.to_vec()))
-                                .ok()
-                                .map(|s| s.swap_id)
-                                .flatten(),
-                        )
-                    })
+            .map(|(key, val)| {
+                Ok((
+                    bitcoin::Address::strict_decode(std::io::Cursor::new(key.to_vec()))?,
```

Comment:
Importing types such as `std::io::Cursor` would make those lines easier to read.

Reply:
We already import `lmdb::Cursor`, so I'll rename it to `IoCursor`.

Reply:
66b3d19f5ed4d907cf6fcc59b26092488f3520b7